cts/cts-exec.in

```diff
@@ -85,18 +86,24 @@ class ExecTest(Test):
 
         self.tls = kwargs.get("tls", False)
 
+        # If we are going to run the stonith resource tests, we will need to
+        # launch and track Corosync and pacemaker-fenced.
+        self._corosync = None
+        self._fencer = None
+        self._is_stonith_test = True if "stonith" in self.name else False
```

Comment:
This could just be `self._is_stonith_test = "stonith" in self.name`.

Reply:
updated